float32[6] th_vel
float32[3] fan_pre
float64 sec_f
int32 steer_state
float32 polish_speed